### PR TITLE
feat(multiple_msg_unread): replace endpoint that marks multiple messa…

### PR DESCRIPTION
**Pull request by hernan604**
…ges as unread/unflag to update these attributes with 1 request only

### CODE REVIEW

File: src/app/rmm/email.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        req.subscribe(
+          data => {
+            const reply = data;
+            if ( reply['status'] === 'error' ) {
+              this.app.show_error( reply['error'].join( '' ), 'Dismiss' );
+              return;
+            }
+          },
+          error => {
+            return this.app.show_error('Could not load aliases.', 'Dismiss');
```
Comment (tadzik) [inline src/app/rmm/email.ts:39]: Wrong error message.

File: e2e/mockserver/mockserver.ts [hunk 2]
```diff
@@ -144,6 +144,11 @@ export class MockServer {
...
+                return;
+            }
```
Comment (tadzik) [inline e2e/mockserver/mockserver.ts:151]: This looks like a leftover from testing and it makes your tests fail (because of double-encoded JSON).

**Comment by tadzik** [comment]
I think the new code doesn't notify the [messageFlagChangeSubject](https://github.com/runbox/runbox7/blob/master/src/app/rmmapi/rbwebmail.ts#L150) (which the messagelistservice observed), so any seen flag updates will only show on the screen after the next index update, possibly making it seem like things aren't working at all.

File: e2e/mockserver/mockserver.ts (hunk 2 above)
Comment (hernan604) [reply e2e/mockserver/mockserver.ts]: removed

**Pull request merged**